src/renderer/components/switch/switch.tsx

```diff
@@ -17,7 +17,13 @@ export function Switch({ children, disabled, onChange, ...props }: SwitchProps)
   return (
     <label className={cssNames(styles.Switch, { [styles.disabled]: disabled })} data-testid="switch">
       {children}
-      <input type="checkbox" role="switch" disabled={disabled} onChange={(event) => onChange?.(props.checked, event)} {...props}/>
+      <input
+        type="checkbox"
+        role="switch"
+        disabled={disabled}
+        onChange={(event) => onChange?.(!props.checked, event)}
```

Comment:
Why not?
```suggestion
        onChange={(event) => onChange?.(event.target.checked, event)}
```

Otherwise if you passing `props.checked` as first argument it's assuming `props.checked` must be required prop for `<Switch/>`? Also it means it's always gonna be only in [controlled react component form](https://reactjs.org/docs/uncontrolled-components.html#:~:text=In%20a%20controlled%20component%2C%20form,form%20values%20from%20the%20DOM.)?